src/client/ui/PropertyGroup.js

```diff
@@ -25,9 +27,17 @@ function PropertyGroup(props) {
 
   const renderSaveButtons = () => {
     if (!saveable) return;
+    const srcStatus = {
+      class: srcIsValid ? styles.src : styles.invalidSrc,
+      tip: srcIsValid ? "file path" : "Cannot find the file.",
```

Comment:
I don't think the "file path" tooltip is necessary when there is no issue with the `src`.

Reply:
Same goes for "file path" in `FileInput` below.